src/store/global/slice.ts

```diff
@@ -81,9 +79,8 @@ const globalSlice = createSlice({
     builder.addCase(fetchStats.rejected, (state) => {
       state.isFetchingStats = false;
     });
-    builder.addCase(fetchUser.fulfilled, (state) => {
-      const tokenParsed = keycloak.tokenParsed as IncludeKeycloakTokenParsed;
-      const locale = tokenParsed.locale || state.lang;
+    builder.addCase(fetchUser.fulfilled, (state, action) => {
```

Comment:
normalement maintenant tu peux retirer completement le `builder.addCase(fetchUser.fulfilled`

Reply:
Je crois pas, j'en ai besoin pour prendre la valeur récupérée du user et la passer à la fonction setLocaleConfig qui s'occupe de la mettre dans le localStorage (qui est la valeur utilisée par le UI)

Reply:
ok ok car on laisse le `setLocalConfig` ici
le petit hic c'est que ca retrigger toujours `setLocalConfig` meme is la locale est deja la bonne donc logiquement ca rerender pour rien.

Au lieu de le faire sur le store global en attendant le store de l'user, à l'inverse, on aurait aussi pu faire un `globalActions.changeLang` sur le user store directement dans son `fulfilled` l.35 de userSlice. Mais ca revient au meme il faut aussi verifier que la locale venant de users-api est difference que celle du store actuel